frontend/src/v5/ui/controls/sideBar/sidebar.component.tsx

```diff
@@ -23,17 +23,15 @@ interface ISidebar {
 	onClick: () => void;
 	className?: string;
 	children: JSX.Element;
-	hidden?: boolean;
 }
 
 export const Sidebar = ({
 	className,
 	open,
 	onClick,
```

Comment:
should we perhaps change the name of this prop so it's clear it refers to clicking the expand button rather than just clicking anywhere on the sidebar